code/game/objects/structures/trash_pile.dm

```diff
@@ -274,6 +274,115 @@
 	else
 		return produce_beta_item()
 
+//// UNDER TETHER TRASH PILE
+
+obj/structure/trash_pile/under_tether
+	name = "old trash pile"
+	desc = "A heap of garbage that has been here for ages, but maybe there's something interesting inside still?"
+	icon = 'icons/obj/trash_piles.dmi'
+	icon_state = "randompile"
+	density = 1
+	anchored = 1.0
+
+	var/obj/structure/mob_spawner/rat_nest/rat_nest = null
+
+/obj/structure/trash_pile/under_tether/Initialize()
+	. = ..()
+	icon_state = pick(
+		"pile1",
+		"pile2",
+		"pilechair",
+		"piletable",
+		"pilevending",
+		"brtrashpile",
+		"microwavepile",
+		"rackpile",
+		"boxfort",
+		"trashbag",
+		"brokecomp")
+	rat_nest = new(src)
+
+/obj/structure/trash_pile/under_tether/Destroy()
+	qdel(rat_nest)
+	rat_nest = null
+	return ..()
+
+/obj/structure/trash_pile/under_tether/attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
is there a reason the attackby/init/destroy procs shared by the parent obj duplicated?

Reply:
Will check this out, might of been duplicated by accident.